src/lib/project.js

```diff
@@ -332,7 +339,42 @@ async function scaffoldNext() {
   }
 };`
   );
+  newNextConfig = newNextConfig.replace(
+    'reactStrictMode: true',
+    'reactStrictMode: false'
+  );
   fs.writeFileSync(path.join('ui', 'next.config.js'), newNextConfig);
+
+  const tsconfig = `
+    {
+      "compilerOptions": {
+        "target": "ES2019",
+        "module": "es2022",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "allowJs": true,
+        "skipLibCheck": true,
+        "strict": true,
+        "strictPropertyInitialization": false, // to enable generic constructors, e.g. on CircuitValue
+        "forceConsistentCasingInFileNames": true,
+        "noEmit": true,
+        "esModuleInterop": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "experimentalDecorators": true,
+        "emitDecoratorMetadata": true,
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "jsx": "preserve",
+        "incremental": true
+      },
+      "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+      "exclude": ["node_modules"]
+    }
+  `;
+
+  if (useTypescript) {
```

Comment:
This will always evaluate to true and execute the code below.  `useTypescript` is always a string of "yes" or "no" which are both truthy. You can do something like if(useTypescript === 'yes');

Reply:
good catch - fixing